src/kcas.ml

```diff
@@ -194,35 +238,43 @@ module Op = struct
   type t = CAS : 'a Loc.t * 'a * 'a -> t
 
   let make_cas loc before after = CAS (loc, before, after) [@@inline]
+  let make_cmp loc expected = CAS (loc, expected, expected) [@@inline]
```

Comment:
There's a few 3-letter abbreviations in the space already. Perhaps `make_compare`?

Reply:
Hmm... I don't have a strong opinion on this, but I kind of like having it the same length with `cas`. 😄

I'm Ok with either `make_cmp` or `make_compare`.

Other votes?